client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -112,6 +113,14 @@ public boolean ping(Header... headers) throws IOException {
     }
 
     /**
+     * Get the cluster info otherwise provided when calling GET `localhost:9200`
+     */
+    public MainResponse info(Header... headers) throws IOException {
```

Comment:
just double checking: when I added ping I thought it doesn't make sense to have an async variant for it, I tend to think the same for info, but let's discuss it, what do others think?